rust-toolchain.toml

```diff
@@ -0,0 +1,5 @@
+[toolchain]
+channel = "nightly"
```

Comment:
Let's a comment here that AquaVM itself could be built with `stable`, but `nightly` is required for Marine tests